app/views/admin/merchants/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Merchant <%= @merchant.id %></h1>
+<%= form_with model: @merchant, url: "/admin/merchants/#{@merchant.id}", method: "patch", local: true do |form| %>
```

Comment:
Correct me if I'm wrong, but I don't believe you have to specify the URL or Method when using 'form_with model'